common/templates/recordset.html

```diff
@@ -12,6 +12,10 @@
                     <button id="search-submit" class="btn btn-primary" ng-click="search(vm.search)" role="button">
                         &nbsp;<span class="glyphicon glyphicon-search"></span>&nbsp;<!-- Added 2 non-breaking spaces to prevent btn's margins from collapsing and to center the icon-->
                     </button>
+                    <button type="button" tabIndex="-1" class="btn btn-primary btn-no-focus coltooltip">
+                        <span class="glyphicon glyphicon-question-sign"></span>
+                        <span class="coltooltiptext paragraph"><p>Use space to separate between conjunctive terms, | (no spaces) to separate disjunctive terms and quotations for exact phrases.</p><p>For example, <i><b>usc 1234</b></i> returns all records containing "usc" and "1234".</p><p><i><b>usc|1234</b></i> returns all records containing "usc" or "1234".</b></i></p><p><i><b>"usc 1234"</b></i> returns all records containing "usc 1234".</span>
```

Comment:
I would just like to mention that we should get rid of these spans and start using Angular Bootstrap tooltips. We already have it in our codebase, we just need to start using them. The syntax is pretty simple.

You can check example [here](https://angular-ui.github.io/bootstrap/#/tooltip). For the above scenario the code would be 


```html
<button type="button" tabIndex="-1" class="btn btn-primary btn-no-focus coltooltip"
tooltip-placement="bottom" tooltip-append-to-body="true"  uib-tooltip-html='<p>Use space to separate between conjunctive terms, | (no spaces) to separate disjunctive terms and quotations for exact phrases.</p><p>For example, <i><b>usc 1234</b></i> returns all records containing "usc" and "1234".</p><p><i><b>usc|1234</b></i> returns all records containing "usc" or "1234".</b></i></p><p><i><b>"usc 1234"</b></i> returns all records containing "usc 1234".'>
       <span class="glyphicon glyphicon-question-sign"></span>
</button>
```